package.json

```diff
@@ -7,6 +7,7 @@
     "lerna": "lerna",
     "bootstrap": "lerna bootstrap",
     "prepare": "if test \"$NODE_ENV\" != \"production\" && test \"$CI\" != \"true\" ; then husky install ; fi",
+    "prepare:pack": "lerna run --stream prepack",
```

Comment:
can be just `prepack`?

Reply:
Not sure, should it run on every prepack cycle?

Reply:
the monorepo root won't have "pack" anything so it's ok to use `prepack`?

Reply:
cool